nsls2forge_utils/xlsheet.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import yaml
+from openpyxl import Workbook
+from openpyxl.styles import PatternFill
+
+wb = Workbook()
+ws = wb.active
+ws.title = "nsls packages data"
+ws.append(
+    [
+        "Repo",
+        "condaforge version",
+        "nsls2forge version",
+        "missing in conda-forge",
+        "PR URLS's",
+        "feedstock URLS",
+        "anaconda.org URLS",
+        "Notes",
+    ]
+)
+redFill = PatternFill(
+    start_color="00FF8080", end_color="00FF8080", fill_type="solid"
+)  # header for chart, for the color 00FF8080: https://openpyxl.readthedocs.io/en/stable/styles.html#indexed-colours
+
+data = []
+
+with open("xldata.yml", "r") as package_data, open(
+    "names.txt", "r"
+) as repos:  # yaml file with package data & names.txt file for i in loop
+    readinfo = yaml.load(package_data, Loader=yaml.FullLoader)
+    data = repos.readlines()
+
+for i in range(2, len(data) + 2):
+    cellref = ws.cell(row=i, column=1)
+    cellref.value = readinfo["package" + str(i - 2)]["package_name"]
```

Comment:
f-strings can be used here for better readability:
```suggestion
    cellref.value = readinfo[f"package{i-2}"]["package_name"]
```